doc/api.rst

```diff
@@ -23,10 +25,9 @@ Utils (:py:mod:`mne_bids.utils`):
    :toctree: generated/
 
    print_dir_tree
-   make_bids_folders
-   make_bids_filename
    make_dataset_description
    copyfile_brainvision
+   copyfile_eeglab
```

Comment:
do you need to make these functions public?

Reply:
at least the `copyfile_brainvision` function is a feature that we also covered in an example: https://mne-tools.github.io/mne-bids/auto_examples/rename_brainvision_files.html#sphx-glr-auto-examples-rename-brainvision-files-py

Reply:
I guess these are useful functions? I don't know ...

Reply:
I don't think they fit on the api page, but I think they should remain public through the utils

Reply:
But we have an example using these functions. Do you propose to remove the example too?